### PR TITLE
Template for Stop too far from trip shape

### PR DESCRIPTION
**Pull request by KathleenQ**
Test is included.
The template looks like:
Screen Shot 2021-02-11 at 2 35 56 pm
The codes have already been run through clang-format.

### CODE REVIEW

File: webapp/src/main/webapp/resources/template.soy [hunk 1]
```diff
@@ -382,4 +382,39 @@
...
+
+/**
+ * Renders a basic explanation of the stop too far from trip shape notice.
+ * @param notices : List<[stopId : string, stopSequence : number, tripId : string, shapeId : string, tripBufferMeters : number]>
+ */
+{template .stopTooFarFromTripShape}
+{let $numNotices: length($notices)/}
+  <p class="error">Error - Stop too far from trip shape!</p>
+  <p>Description: Stop is too far away from the trip shape.</p>
+  <p><b>{$numNotices}</b> Stop(s) too far from trip shape found in:</p>
```
Comment (lilian-hunt) [inline webapp/src/main/webapp/resources/template.soy]: This doesn't make sense grammatically. I also think there is too much repetition, so it is sufficient to say 
`<p><b>{$numNotices}</b> found:</p>`

**Review by aannie-y — COMMENTED** [review]
besides what lilian said, lgtm

File: webapp/src/main/webapp/resources/template.soy [hunk 2]
```diff
@@ -382,4 +382,39 @@
...
+          <td>{$notice.stopId}</td>
+          <td>{$notice.stopSequence}</td>
+          <td>{$notice.tripId}</td>
+          <td>{$notice.shapeId}</td>
+          <td>{$notice.tripBufferMeters}</td>
+        </tr>
+      {/foreach}
+    </tbody>
+  </table>
+  <p>Please fix the stop position to make it closer than the trip buffer meters from the trip shape!</p>
```
Comment (aannie-y) [inline webapp/src/main/webapp/resources/template.soy]: Bit confusing. Maybe word it like "Please fix the stop position to be within the trip buffer metres of the trip shape"

File: webapp/src/main/webapp/resources/template.soy (hunk 1 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Yes. Have adjusted all templates here.

File: webapp/src/main/webapp/resources/template.soy (hunk 2 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Yes. That sounds better! I have already adjusted it.

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**